src/foam/u2/view/LoginView.js

```diff
@@ -242,7 +259,7 @@ foam.CLASS({
       // CREATE MODEL VIEW
       var right = this.E()
       // Header on-top of rendering model
-        .start().hide(this.imgPath$).addClass('topBar-logo-Back')
+        .start().show(this.imgPath$).addClass('topBar-logo-Back')
```

Comment:
Why this change?

Reply:
think i might have misunderstood what this was supposed to do, will change back, thanks